src/Host/Client/Impl/DataInspection/REnvironments.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.R.DataInspection {
+    /// <summary>
+    /// Expressions for various common R environments, suitable for use as <c>environmentExpression</c> argument
+    /// when calling <see cref="RSessionExtensions.EvaluateAndDescribeAsync"/>.
+    /// </summary>
+    public static class REnvironments {
```

Comment:
Probably. When you're using regular `EvaluateAsync`, you don't specify the target environment as an expression, but rather use `REvaluationKind.BaseEnv` etc. I guess it may occasionally be useful for when your evaluated expression needs to incorporate a reference to an environment somewhere. 

Where would you prefer it to be?